src/EnergyPlus/HVACStandAloneERV.cc

```diff
@@ -2047,4 +2048,91 @@ namespace EnergyPlus::HVACStandAloneERV {
         return GetStandAloneERVReturnAirNode;
     }
 
+    bool GetStandAloneERVNodeNumber(EnergyPlusData &state, int const NodeNumber)
+    {
+        // PURPOSE OF THIS FUNCTION:
+        // Check if a node is used by a stand alone ERV
+        // and can be excluded from an airflow network.
+
+        // Return value
+        bool StandSloneERVAFNException;
+
+        int StandAloneERVIndex;
```

Comment:
On brand new code, there's no reason to follow these fortran patterns of declaring the variables up front.  Not going to hold this up for now, but just noting for the future.